### ISSUE TITLE
Experiments name and length 

### ISSUE THREAD

**Issue by pmathiot** [jbbarre/ISM_SimulationChecker#5]
I wanted to try the tools with our first complete simulation and I figure out that the list of experiments does not match : https://www.climate-cryosphere.org/wiki/index.php?title=ISMIP6-Projections2300-Antarctica

When you do this, maybe clarify exactly what name need to be used (expAE04 as in the table or AE04 as in section A2.1 in the exemple) ?

**Comment by jbbarre** [comment]
You're right @pmathiot , there is an unconstistancy between the ISMIP6_EXP_AIS_2300 table and the naming convention in section A2.1( <EXP> = experiment name (from the experiment list, e.g. AE01)). I suggest to keep the _exp_ part as in ISMIP6. If ok, the example must be modified : <EXP> = experiment name (from the experiment list, e.g. **exp**AE01).
@hseroussi, Do you confirm ?

**Comment by pmathiot** [comment]
I retest it and there is an issue with the error message. I kept my old name exp from yesterday test (exp03) and I have an error message say that:

```
ERROR: The compliance check is ignored for experiment exp03 as it is not in [hist, ctrl, ctrl_proj, exp01, exp02, exp03, exp04, exp05, exp06, exp07, exp08, exp09, exp10, exp11, exp12, exp13].
```

This is the wrong exp name list. I should have the name list you added yesterday ...

I suggest in l494 to use [dexp['experiment'] for dexp in experiments] instead of the hard coded list (suggestion not tested).

**Comment by hseroussi** [comment]
Thank you for pointing this out. I changed the text on the wiki to be consistent with the table, so names should be expAE01, etc. 

For the script, the default experiments in the tests are the old experiments so people can first check their installation with the test file, then they have to change the list of experiments in the compliance script to switch to the experiments_ismip6_ext list. I wrote that in the README file but we change change it if you think it's not clear.
